apps/generic-issuance-client/src/pages/pipeline/DetailsSections/PipelinePCDMetadata.tsx

```diff
@@ -0,0 +1,71 @@
+import { CheckIcon, CopyIcon } from "@chakra-ui/icons";
+import { Button, useToast } from "@chakra-ui/react";
+import { PipelinePCDMetadata } from "@pcd/passport-interface";
+import { ReactNode, useCallback, useState } from "react";
+
+/**
+ * Used to display metadata about the PCDs produced by the pipeline.
+ */
+export function PipelinePCDMetadataSection({
```

Comment:
Yeah, I did consider changing it to that and wasn't sure if it was too narrow (I could imagine other uses for the same data). But it would more accurately describe what it's used for now.